### PR TITLE
Note python3 document

### PR DESCRIPTION
**Pull request by cancan101**

### CODE REVIEW

**Comment by taion** [comment]
I would say this isn't worth adding, just on the principle of "obviousness".

File: README.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ This guide is intended to present general guidelines. Most modules should follow
 
 Whenever possible, projects should use the newest release of Python 3.
 
+### Features
+
+Whenever possible, projects should take advantake of Python 3 features and syntax. For example prefer [new style super calls](https://docs.python.org/3.5/library/functions.html#super) to old stlye.
```
Comment (nevsan) [inline README.md]: `advantake` => `advantage`

**Comment by nevsan** [comment]
A lot of people don't do this and I have to remind them in PRs. I view this as a good place to start collecting other Py3 goodies devs may not know about (like `a // b` instead of `int(a/b)`).

**Comment by taion** [comment]
Right – I think it could be good to have a "Python 3 for Python 2 devs" guide. The style guide proper isn't the right place, though, since this isn't a matter of style.

**Comment by cancan101** [comment]
@taion Where do you suggest such a document lives?
I think there will also exist a set of practices that we want to discourage (similar to this) that are not specific to the py2 to py3 transition. For example preferring `tuple` literals over `list` literals.

**Comment by taion** [comment]
Tuple v list belongs in a style guide.

If you want to make a "Python 3 guide", go ahead.

**Comment by nevsan** [comment]
How about we at least have a note here to observe recommendations in the "Python 3 guide" (and [link](https://github.com/blog/1395-relative-links-in-markup-files) to it).

**Comment by itajaja** [comment]
btw, clearly some better automated linting would solve the problem here...

**Comment by cancan101** [comment]
I opened: https://github.com/4Catalyzer/python/pull/6

**Comment by cancan101** [comment]
will update this to be a relative link to the new document

**Comment by taion** [comment]
No – do this all in #6; it's an atomic change to "add the Python 3 guide and link to it".

File: README.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ This guide is intended to present general guidelines. Most modules should follow
 
 Whenever possible, projects should use the newest release of Python 3.
 
+Whenever possible, projects should take advantage of Python 3 features and syntax. [Follow our document on features to use in Python 3](Python3.md).
```
Comment (itajaja) [inline README.md]: sounds redundant with previous sentence. merge them?

**Pull request merged**